### PR TITLE
6.9.1 website changes

### PR DESCRIPTION
**Pull request by sanjanasandeep**

### CODE REVIEW

File: content/en/get-started/cards/docker-691.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+### CDAP 6.9.1 for Docker
+
+#### Description
+
+CDAP Docker provides a clean, contained environment for trying out CDAP on your desktop or laptop.
+
+####  What's New:
+
+* View [release notes](https://cdap.atlassian.net/wiki/spaces/DOCS/pages/1888059393/CDAP+Release+6.9.1)
```
Comment (sanjanasandeep) [inline content/en/get-started/cards/docker-691.md:9]: @CuriousVini is this correct?

Comment (CuriousVini) [reply content/en/get-started/cards/docker-691.md:9]: Use this one

https://cdap.atlassian.net/wiki/spaces/DOCS/pages/1902084097/CDAP+Release+6.9.1

File: data/en/get_started.json [hunk 2]
```diff
@@ -1380,6 +1527,92 @@
...
+            {
+              "id": "vm-691",
+              "item": {
+                "title": "CDAP for VM",
+                "img": "/images/get-started/vm.svg",
+                "buttons": [
+                  {
+                    "buttonTitle": "CDAP 6.9.1 for VM",
+                    "id": "version-691-vm",
+                    "buttonHref": "https://github.com/cdapio/cdap-build/releases/download/v6.9.1/cdap-sandbox-6.9.1.ova"
```
[the remark this answers is not in the document]
Comment (CuriousVini) [reply data/en/get_started.json:1539]: Hmm looks like this link has been broken for all the releases:
https://github.com/cdapio/website/pull/202/files#diff-825162c53680862866c6bd274c78a76f718f3d779eadb73cc8f304205d8a539aR1474

**Comment by sanjanasandeep** [comment]
conflict

**Pull request closed without merging**